content/en/docs/Installing and Configuring DreamFactory/_index.md

```diff
@@ -211,6 +211,18 @@ Immediately following this prompt you'll be informed of successful setup:
 	* your instance using a browser. You can run "php artisan serve"
 	* to try out your instance without setting up a web server.
 	**********************************************************
+After creating an Admin, your instance may show the following error:
+```
+Admins
+There is no root administrator selected. Some functionality might not work. Use df:root_admin command to choose one.
+```
+
+A very simple approach to solve it, is to reset the root from inside your container:
+`$ php artisan df:root`
+`$ php artisan migrate:fresh --seed`
```

Comment:
Are you sure this is necessary? It seems a very bad idea because it will drop all tables, and if the customer has users and settings it will delete everything.